README.adoc

```diff
@@ -240,6 +240,53 @@ spec:
               fieldPath: status.hostIP
 ----
 
+== Schema migration
+
+=== Cassandra
+
+When the storage type is set to Cassandra, the operator will automatically create a batch job that creates the required schema for Jaeger to run. This batch job will block the Jaeger installation, so that it starts only after the schema is successfuly created. The creation of this batch job can be disabled by setting the `enabled` property to `false`:
```

Comment:
Just wondering if it would be safer to default to false? If someone is using an existing jaeger cassandra storage - what is the impact of the job running, is it a noop?

Reply:
The current scripts are noop, as they have an "if not exists" clause:

https://github.com/jaegertracing/jaeger/blob/7d9d212b6d947742055fb4e38180152da687acf5/plugin/storage/cassandra/schema/v001.cql.tmpl#L25

But it might be a good idea to add a flag to the binary, to get this enabled/disabled by default, as we discussed today in the meeting. Something for a follow up PR, to be discussed with the ES parts?